library/src/main/java/com/github/pwittchen/prefser/library/PreferencesAccessorsProvider.java

```diff
@@ -48,7 +48,7 @@ private void createAccessors() {
   private void createBooleanAccessor() {
     accessors.put(Boolean.class, new Accessor<Boolean>() {
       @Override public Boolean get(String key, Boolean defaultValue) {
-        return preferences.getBoolean(key, defaultValue);
+        return preferences.getBoolean(key, (defaultValue == null) ? false : defaultValue);
```

Comment:
Why would user pass the `null`? Why should the user expect `false` in the case of passing `null`, but not `true`? How could he or she guess it just by looking at the library API?